client/components/Game.js

```diff
@@ -78,11 +78,19 @@ class Game extends React.Component {
                             </Card>
                         </Col>
                     </Row>
+                    <div className="host-notification">
+                        {(this.props.game.host === this.props.player._id) ? <div> YOU ARE THE HOST! </div> : <div> Please submit a card </div>}
```

Comment:
Consider changing the wording. Instead of you are the host, maybe print "Your turn". Just don't use the word host